redux/auth/actions.test.ts

```diff
@@ -57,4 +79,77 @@ describe(`${DOMAIN_NAME}/actions`, () => {
 
     expect(result).toEqual(expected);
   });
+
+  test('signup action creator returns correct object', () => {
+    const mockedEmail: string = 'mocked-email@email.com';
+    const mockedPassword: string = 'mocked-password';
+
+    const result = signup(mockedEmail, mockedPassword);
+
+    const expected: SignupAction = {
+      type: SIGNUP,
+      payload: { email: mockedEmail, password: mockedPassword },
+    };
+
+    expect(result).toEqual(expected);
+  });
+
+  test('signupSuccess action creator returns correct object', () => {
+    const mockedToken: string = 'mocked-secure-token';
+    const mockedUserData = {
+      name: 'user-name',
+      id: 'user-id',
+      image: 'user-image-url',
+    };
+
+    const result = signupSuccess(mockedToken, mockedUserData);
+
+    const expected: SignupSuccessAction = {
+      type: SIGNUP_SUCCESS,
+      payload: { token: mockedToken, userData: mockedUserData },
+    };
+
+    expect(result).toEqual(expected);
+  });
+
+  test('signupFailure action creator returns correct object', () => {
+    const mockedMessage = 'mocked-message';
+
+    const result = signupFailure(mockedMessage);
+
+    const extected: SignupFailureAction = {
+      type: SIGNUP_FAILURE,
+      payload: mockedMessage,
+    };
+
+    expect(result).toEqual(extected);
+  });
+
+<<<<<<< Updated upstream
```

Comment:
Merge marker. They are now in a couple of places. Please fix it.

Reply:
Done :+1: